vector/src/main/java/im/vector/app/core/extensions/Session.kt

```diff
@@ -34,9 +34,21 @@ fun Session.configureAndStart(context: Context, startSyncing: Boolean = true) {
         startSyncing(context)
     }
     refreshPushers()
+    configureContentScanner()
     context.singletonEntryPoint().webRtcCallManager().checkForProtocolsSupportIfNeeded()
 }
 
+private fun Session.configureContentScanner() {
+    sessionParams.homeServerConnectionConfig.antiVirusServerUri
+            ?.let { url ->
```

Comment:
`?.let` is not useful here (we do not use the resulting value, only check the nullability), so I would prefere to declare a variable and then use a simple if to check the nullability or trigger a warning

Reply:
It used at line 46
`it.setScannerUrl(url.toString())`